apps/src/codebridge/FileBrowser/PopUpButtonOption.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+
+import moduleStyles from './styles/filebrowser.module.scss';
+import darkModeStyles from '@codebridge/styles/dark-mode.module.scss';
+
+type PopUpButtonOptionProps = {
+  iconName: string;
+  labelText: string;
+  clickHandler?: () => void;
+};
+
+export const PopUpButtonOption = ({
```

Comment:
Mind adding a comment on this component? I think this also generic enough that it could live beside `PopUpButton` in the `PopUpButton` folder.

Reply:
Done!